data/metadataSchema.json

```diff
@@ -0,0 +1,163 @@
+  "title": "Field Metadata for CSAF Documents",
+  "description": "JSON schema for json file storing metadata on fields of CSAF documents",
+  "type": "object",
+  "$defs": {
+    "relevance_levels_t": {
+      "title": "Relevance Levels",
+      "description": "Defines the possible relevance levels",
+      "type": "string",
+      "enum": [
+        "mandatory",
+        "best_practice",
+        "want_to_have",
+        "nice_to_know",
+        "optional",
+        "excluded"
+      ]
+    },
+    "documentation_file_path_t": {
+      "title": "File Path",
+      "description": "Defines the path to a file on the file system",
+      "type": "string",
+      "pattern": "docs/user(/.+)+-(spec|usage)(-csaf_(base|security_incident_response|informational_advisory|security_advisory|vex))?\\.[a-z]{2}\\.md"
+    }
+  },
+  "properties": {
+    "metaData": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "path",
+          "user_documentation",
+          "description",
+          "label"
+        ],
+        "properties": {
+          "path": {
+            "title": "Json Path",
+            "description": "The json path of the field this item holds metadata for",
+            "type": "string",
+            "pattern": "\\$(\\.([A-Za-z_](\\[\\])?)+)+"
+          },
+          "relevance_levels": {
+            "title": "Relevance Levels",
+            "description": "Relevance levels of the field per profile",
+            "type": "object",
+            "properties": {
+              "csaf_base": {
+                "title": "CSAF Base Relevance Levels",
+                "description": "Relevance Levels for the CSAF Base profile",
+                "$ref": "#/$defs/relevance_levels_t"
+              },
+              "csaf_informational_advisory": {
+                "title": "CSAF Informational Advisory Relevance Levels",
+                "description": "Relevance Levels for the CSAF Informational Advisory profile",
+                "$ref": "#/$defs/relevance_levels_t"
+              },
+              "csaf_security_advisory": {
+                "title": "CSAF Security Advisory Relevance Levels",
+                "description": "Relevance Levels for the CSAF Security Advisory profile",
+                "$ref": "#/$defs/relevance_levels_t"
+              },
+              "csaf_security_incident_response": {
+                "title": "CSAF Security Incident Relevance Levels",
+                "description": "Relevance Levels for the CSAF Security Incident profile",
+                "$ref": "#/$defs/relevance_levels_t"
+              },
+              "csaf_vex": {
+                "title": "CSAF Vex Relevance Levels",
+                "description": "Relevance Levels for the CSAF Vex profile",
+                "$ref": "#/$defs/relevance_levels_t"
+              }
+            },
+            "additionalProperties": false
+          },
+          "description": {
```

Comment:
Please try to avoid (a raw) `description` as it is a JSON schema keyword. Maybe `field_description` could work? But I'm open to better suggestions.

Reply:
Good point. Your suggestion is fine, I updated it.